src/status_im/contexts/onboarding/identifiers/view.cljs

```diff
@@ -22,7 +22,7 @@
 
 (defn- navigate-to-enable-notifications
   []
-  (rf/dispatch [:onboarding/navigate-to-enable-notifications]))
+  (rf/dispatch [:onboarding/navigate-to-enable-notifications :screen/onboarding.identifiers]))
```

Comment:
could you elaborate?

Reply:
Well this screen is not needed we decided to keep the file. I changed the code here for completion.

Reply:
it should be `screen/onboarding.generating-keys` not `:screen/onboarding.identifiers`. I have updated that